package.json

```diff
@@ -38,11 +38,10 @@
     "testURL": "http://localhost"
   },
   "dependencies": {
+    "@storybook/components": "^5.2.5",
     "copy-to-clipboard": "^3.0.8",
     "js-beautify": "^1.8.8",
-    "prismjs": "^1.15.0",
-    "react-element-to-jsx-string": "^14.0.3",
-    "storybook-chromatic": "^3.0.3"
```

Comment:
Is this even needed? doesn't seem to be used anywhere

Reply:
It's used to run chromatic locally, to set it up, can be removed once it's automated in a CI / action (which is now is)